options/internal/include/bits/nl_item.h

```diff
@@ -102,8 +102,6 @@ enum { /* NOLINT(readability-enum-initial-value) */
 	_NL_CTYPE_EXTRA_MAP_12,
 	_NL_CTYPE_EXTRA_MAP_13,
 	_NL_CTYPE_EXTRA_MAP_14,
-	_NL_CTYPE_EXTRA_MAP_15,
-	_NL_CTYPE_EXTRA_MAP_16,
 	_NL_NUM_LC_CTYPE,
```

Comment:
The commit message says that this is not an ABI break but this changes the value of `_NL_NUM_LC_CTYPE`.